tests/integration/kubernetes/tests_common.sh

```diff
@@ -166,8 +169,7 @@ auto_generate_policy() {
 	declare -r config_map_yaml_file="$3"
 
 	auto_generate_policy_enabled || return 0
-
-	local genpolicy_command="RUST_LOG=info /opt/kata/bin/genpolicy -u -y ${yaml_file}"
+	local genpolicy_command="RUST_LOG=info /opt/kata/bin/genpolicy -u -d -y ${yaml_file}"
```

Comment:
Please add a test that doesn't use the new -d parameter.

Examples of similar one-off tests for Policy are: 
- k8s-exec-rejected.bats
- k8s-policy-set-keys.bats

I believe these get executed just on platforms that support Policy.

Reply:
Thinking more about this, what about making `k8s-pod-manifest-v1.bats` the exception test that uses `-d` and keep the rest of the tests using default pull. It's still the default option and will probably be still more commonly used

Reply:
These genpolicy tests are currently enabled just for Mariner hosts. Would running all of the tests twice - with and without -d - work?

Reply:
Ended up adding a [single test scenario](https://github.com/kata-containers/kata-containers/pull/9185/files#diff-55a1b92d4c9735afcdbe6ef38905599f0f58dbdd7cd5f8db16333529d57a47c3R159) rerun using default pull method